includes/tools.php

```diff
@@ -223,3 +226,145 @@ function verify_https_support() {
 	 */
 	return 200 === wp_remote_retrieve_response_code( $response );
 }
+
+/**
+ * Build a system environment report for submission along with any bug reports.
+ *
+ * This is largely based on (a simplified version) of Easy Digital Downloads' edd_tools_sysinfo_get()
+ * reporting function.
+ *
+ * @link https://github.com/easydigitaldownloads/easy-digital-downloads
```

Comment:
Reached out to @pippinsplugins via Twitter DM to get his input, since he's the original copyright holder as owner of @easydigitaldownloads. I'd feel more comfortable getting his blessing before merging this + keeping MIT.